selfservice/strategy/code/code_sender.go

```diff
@@ -0,0 +1,220 @@
+package code
```

Comment:
This file was renamed and then modified beyond recognition, so GitHub doesn't see that they're the same file. Best to diff it locally @reviewer.